migration/migration-technical-details.md

```diff
@@ -4,17 +4,21 @@ Nimiq is transitioning from a Proof-of-Work (PoW) to a Proof-of-Stake (PoS) bloc
 
 Each phase operates within a specified block window, enabling transactions to be sent to the network at specific points within the PoW chain.
 
-Our approach consists of targeting a specific block height within the PoW chain designated as the 'transition block.' At this block height, the blockchain state will be captured and used to generate the genesis block for the PoS chain. Instead of starting from block height 0, both chains will share the same block number.
+Our approach consists of targeting a specific block height within the PoW chain designated as the *transition block*. At this block height, the blockchain state will be captured and used to generate the "genesis block" for the PoS chain. Instead of starting from block height 0, both chains will share the same block number. The PoS chain starts once the conditions covered in this document are met.
```

Comment:
Instead of starting from block height 0, the PoS chain will start from the same block number as the transition block, if the conditions covered in this document are met